plugins/terminal/terminal.plugin.desktop.in

```diff
@@ -0,0 +1,12 @@
+[Plugin]
+Loader=python3
+Module=terminal
+IAge=3
+Name=Embedded Terminal
+Description=Embed a terminal in the bottom pane.
+# Translators: Do NOT translate or transliterate this text (this is an icon file name)!
+Icon=utilities-terminal
+Authors=Paolo Borelli
+Copyright=Copyright © 2005 Paolo Borelli
+Website=http://www.gedit.org
```

Comment:
I suppose we have here a forked version, I think it makes sense to have mate url instead